config.json

```diff
@@ -547,24 +547,24 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-auth",
-        "version": "20.0.1",
-        "nugetVersion": "120.0.1",
+        "version": "20.0.2",
+        "nugetVersion": "120.0.2",
         "nugetId": "Xamarin.Firebase.Auth",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-auth-interop",
-        "version": "19.0.1",
-        "nugetVersion": "119.0.1",
+        "version": "19.0.2",
+        "nugetVersion": "119.0.2",
         "nugetId": "Xamarin.Firebase.Auth.Interop",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-common",
         "version": "19.5.0",
-        "nugetVersion": "119.5.0",
+        "nugetVersion": "1119.5.0",
```

Comment:
I think this is the only incorrect one now.  😉

Reply:
Fixed